src/sentry/event_manager.py

```diff
@@ -2637,6 +2666,12 @@ def save_transaction_events(jobs: Sequence[Job], projects: ProjectsMapping) -> S
     with metrics.timer("save_transaction_events.send_occurrence_to_platform"):
         _send_occurrence_to_platform(jobs, projects)
 
+    with metrics.timer("save_transaction_events.sample_transactions"):
+        _sample_transactions_in_save(jobs, projects)
+
+    with metrics.timer("save_transaction_events.send_transaction_processed_signals"):
+        _send_transaction_processed_signals(jobs, projects)
```

Comment:
Should we wrap this in a `try ... except` and log the exception? Previously `save_transaction_events` would not fail if any if this logic failed.

Reply:
will wrap in try/catches. thanks